webapp/components/Editor/Editor.vue

```diff
@@ -205,10 +205,13 @@ export default {
       if (!query) {
         return items
       }
-      return items.filter(item => {
+
+      items = this.suggestionType === HASHTAG ? this.hashtags : this.users
+      const filteredList = items.filter(item => {
         const itemString = item.slug || item.id
         return itemString.toLowerCase().includes(query.toLowerCase())
       })
+      return filteredList.slice(0, 14)
```

Comment:
Is there a reason why you used `14` here and `15` above? I'd say let's maybe use the same number in both places? :smile:

Reply:
haha, just cause I originally had all as 14, but then forgot to change this one